crates/python/src/lib.rs

```diff
@@ -50,3 +50,39 @@ create_init_submodule! {
 fn qcs_sdk(py: Python<'_>, m: &PyModule) -> PyResult<()> {
     init_submodule("qcs_sdk", py, m)
 }
+
+pub(crate) mod utils {
+    /// Spawn and block on a future using the pyo3 tokio runtime.
+    /// Useful for returning a synchronous `PyResult`.
+    ///
+    ///
+    /// When used like the following:
+    /// ```rs
+    /// async fn say_hello(name: String) -> String {
+    ///     format!("hello {name}")
+    /// }
+    ///
+    /// #[pyo3(name="say_hello")]
+    /// pub fn py_say_hello(name: String) -> PyResult<String> {
+    ///     py_sync!(async move {
+    ///         say_hello(name)
+    ///     })
+    /// }
```

Comment:
I feel like this should be one of these:

```suggestion
    /// #[pyo3(name="say_hello")]
    /// pub fn py_say_hello(name: String) -> PyResult<String> {
    ///     py_sync!(async move {
    ///         say_hello(name).await
    ///     })
    /// }
```

```suggestion
    /// #[pyo3(name="say_hello")]
    /// pub fn py_say_hello(name: String) -> PyResult<String> {
    ///     py_sync!(say_hello(name))
    /// }
```

Reply:
Agreed, and the latter seems cleaner